skimage/restoration/_cycle_spin.py

```diff
@@ -145,6 +145,16 @@ def _run_one_shift(shift):
         tmp = func(xs, **func_kw)
         return _roll_axes(tmp, -np.asarray(shift))
 
+    if num_workers != 1:
+        try:
+            import dask
+        except ImportError:
+            import warnings
```

Comment:
This looks like a duplicate import.

Reply:
+1 @yarikoptic Once you remove this line, it'll be ready to be merged and backported.